### PR TITLE
WRR-6681: Fixed typos and JSDoc type

### PR DESCRIPTION
**Pull request by daniel-stoian-lgp**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
There were multiple typos or grammar errors in the enact repository code.
There was a mismatch between the `type` of a JSDoc and the used values in packages/ui/Cancelable/Cancelable.js

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Fixed typos and JSDoc type

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRR-6681

### Comments

Enact-DCO-1.0-Signed-off-by: Daniel Stoian ([daniel.stoian@lgepartner.com](mailto:daniel.stoian@lgepartner.com))

### CODE REVIEW

**Comment by seunghoh** [comment]
Hawkeye 👍

**Review by stanca-pop-lgp — APPROVED** [review]
Good job!!

File: packages/ui/resolution/resolution.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ let baseScreen,
 	config;
 
 /**
- * Object that stores all of the pixel conversion factors to each keyed unit.
+ * Object that stores all the pixel conversion factors to each keyed unit.
```
Comment (juwonjeong) [inline packages/ui/resolution/resolution.js:21]: How about fixing it like this?
```suggestion
 * Object that stores the pixel conversion factors to each keyed unit.
```

**Review by juwonjeong — APPROVED** [review]
LGTM

**Pull request merged**